plugins/languages/markdown-preview.nix

```diff
@@ -0,0 +1,126 @@
+{ config, pkgs, lib, ... }:
+with lib;
+let
+  cfg = config.plugins.markdown-preview;
+  helpers = import ../helpers.nix { inherit lib; };
+in
+{
+  options = {
+    plugins.markdown-preview = {
+      enable = mkEnableOption "Enable markdown-preview";
```

Comment:
Should just be "markdown-preview" without enable

Reply:
Done :)